### DISCUSSION TITLE
Value skip

### DISCUSSION THREAD

**Discussion by popkovanton** [patrykandpatrick/vico#707] [Questions] [answered]
How can I make a value skip on a line?
<img width="591" alt="Screenshot 2024-05-16 at 10 52 58" src="https://github.com/patrykandpatrick/vico/assets/19818752/128704ab-f6c4-41cb-b381-91b74ae07079">

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can do this by specifying the _x_ values explicitly and skipping the _x_ value in question. In the simple example below, _x_ = 2 is skipped. Please see [here](https://patrykandpatrick.com/vico/wiki/cartesian-charts/layers/line-layer#data) for more.

```kt
series(x = listOf(0, 1, 3), y = /* ... */)
```
